api/v1/utils/check_in.js

```diff
@@ -0,0 +1,18 @@
+
+var _ = require('lodash');
+
+var LOCATIONS = ['NONE', 'ECEB', 'SIEBEL', 'DCL'];
+
+/**
+ * Ensures that location upon check in is valid
+ * @param {String} location the check in location
+ * @return {Boolean} true when the location is valid
+ * @throws TypeError when the location is invalid
+ */
+module.exports.verifyLocation = function (location) {
+    if (location && !_.includes(LOCATIONS, location)) {
```

Comment:
You should be fine doing `if (!_.includes(LOCATIONS, location);`, right now if location is `Nil` `location` would evaluate to false and you'd incorrectly return true.